### PR TITLE
v2 api for interaction with credit cards

### PR DESCRIPTION
**Pull request by ba1ash**
`post v2/credit_cards` and `put  v2/credit_cards/:token` is handled

### CODE REVIEW

File: lib/be_gateway/client.rb [hunk 1]
```diff
@@ -37,15 +36,24 @@ def create_card_by_token(token, params)
       make_response(response)
     end
 
+    def v2_create_card(params)
+      response = post '/v2/credit_cards', { request: params }
+      make_response(response)
+    end
+
+    def v2_create_card_by_token(token, params)
+      response = put "/v2/credit_cards/#{token}", { request: params }
```
Comment (seongreen) [inline lib/be_gateway/client.rb]: may be update ?

Comment (ba1ash) [inline lib/be_gateway/client.rb]: It makes sense. Updated.

**Comment by ba1ash** [comment]
:up:

File: lib/be_gateway/version.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module BeGateway
-  VERSION = '0.10.0'
+  VERSION = '0.11.0'
```
Comment (shiroginne) [inline lib/be_gateway/version.rb]: since this PR brings new functionality, I would recommend to up to 0.2.0

**Comment by shiroginne** [comment]
:shipit:

**Comment by seongreen** [comment]
:ok:

**Comment by iermakovets** [comment]
:ok:

**Pull request merged**